pages/[resourceType]/[id]/index.tsx

```diff
@@ -27,6 +29,32 @@ function ResourceIDPage() {
         })
         .then((resourcePageBody) => {
           setPageBody(JSON.stringify(resourcePageBody, null, 2));
+          console.log("resourceType: ", resourceType);
+          //if resource is a measure, an evaluate measure button will be rendered
+          if (resourceType === "Measure") {
+            setEvaluateMeasureButton(
+              <Link
+                href={`/${resourceType}/${id}/evaluate`}
+                key={`evaluate-measure-${id}`}
+                passHref
+              >
+                <Button
+                  component="a"
+                  color="cyan"
+                  radius="md"
+                  size="sm"
+                  variant="filled"
+                  style={{
+                    float: "right",
+                    marginRight: "8px",
+                    marginLeft: "8px",
+                  }}
+                >
+                  <div>Evaluate Measure</div>
+                </Button>
+              </Link>,
+            );
+          }
```

Comment:
we can get rid of this whole block and move some of it to directly in the component, see below